keps/sig-node/127-user-namespaces/README.md

```diff
@@ -250,6 +312,56 @@ limit the number of pods using user namespaces to `min(maxPods, 1024)`. This
 leaves us plenty of host UID space free and this limits is probably never hit in
 practice. See UNRESOLVED for more some UNRESOLVED info we still have on this.
 
+##### pkg/volume changes for phase I
```

Comment:
Comment to line 308 (follow up from your response to my comment):

> The UID they choose is inside the user namespace, it doesn't affect the mapping picked on the host.
> 
> Since we are always allocating the same size, there is no fragmentation to worry about (in facts we just use a bitmap to store at runtime what ranges are allocated).
>
> The picked range is stored in the file /var/lib/kubelet/pods/$POD/userns so the Kubelet can read all the allocated mappings if it restarts

Can you please add that information to the doc - I think it's actually a useful context for people who are not that familiar with details like me.

Reply:
Yes, good call. Thanks!

I've kept this as a different commit, as I think it might be way simpler in this case to review as a separate commit. The commit title is: KEP-127: Clarify how IDs will be used and stored